test/unit/argument_parser/detail/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
+add_definitions(-DSEQAN_INCLUDE_DIR="${SEQAN3_INCLUDE_DIR}")
```

Comment:
why was this change necessary?

Reply:
see the commit message. Because we want to read the licence file (and therefore we need the repo path) in order to compare it to the hard-coded copyright printing and keep things updated.